torch_complex/functional.py

```diff
@@ -46,7 +46,7 @@ def einsum(equation, *operands):
     >>> numpy.testing.assert_allclose(test.numpy(), valid)
 
     """
-    if len(operands) == 1 and isinstance(operands, (tuple, list)):
+    if len(operands) == 1:
```

Comment:
Sorry, I didn't check your modification.

Einsum also supports a single argument.

The second part in my if statement was wrong.
It should check it the entry in the tuple is a tuple or list (Not if the tuple is a tuple or list).
i.e. `if len(operands) == 1 and isinstance(operands[0], (tuple, list)):` (I forgot the `[0]`)

The reason, why this is necessary, is that torch took the same function name as numpy, but initially they had a different signature.
At some point (Not sure, when) they allowed also the numpy style call.

Reply:
I see. Could you fix again?

Reply:
Sure, I will make a new PR.